### PR TITLE
DOCS-823: Add yahboom-dofbot configuration info

### PR DESCRIPTION
**Pull request by sguequierre**
* This model is MVP but had escaped documentation until now- @dannenberg pointed out and found config 
* Config needs to be tested for updated accuracy. I think i2c attribute is pretty consistent across models so should be ok without but there do appear to be some current issues w/ the driver

### CODE REVIEW

File: docs/components/arm/yahboom-dofbot.md [hunk 1]
```diff
@@ -3,35 +3,85 @@ title: "Configure a yahboom-dofbot Arm"
...
 type: "docs"
-draft: "true"
 description: "Configure a yahboom-dofbot arm."
 images: ["/components/img/components/arm.svg"]
 tags: ["arm", "components"]
-# SMEs: William Spies
 ---
 
-Configure a `yahboom-dofbot` arm to integrate the [Yahboom DOFBOT](https://category.yahboom.net/collections/r-robotics-arm) into your robot.
+Configure an `yahboom-dofbot` arm to add a [Yahboom DOFBOT](https://category.yahboom.net/collections/r-robotics-arm) to your robot:
```
Comment (andf-viam) [inline docs/components/arm/yahboom-dofbot.md:11]:
```suggestion
Configure a `yahboom-dofbot` arm to add a [Yahboom DOFBOT](https://category.yahboom.net/collections/r-robotics-arm) to your robot:
```

File: docs/components/arm/yahboom-dofbot.md [hunk 2]
```diff
@@ -3,35 +3,85 @@ title: "Configure a yahboom-dofbot Arm"
...
+        "local"
+      ]
     }
+  ]
 }
 ```
 
 {{% /tab %}}
-{{< /tabs >}}
+{{% /tabs %}}
```
Comment (andf-viam) [inline docs/components/arm/yahboom-dofbot.md:81]: I mostly see `{{< /tabs >}}` in the docs, not `{{% /tabs %}}`. But it builds and works!

However, if you start the tabset definition with `{{< tabs >}}` (line 13), let's end it with a matching `{{< /tabs >}}`

File: docs/components/arm/yahboom-dofbot.md [hunk 3]
```diff
@@ -3,35 +3,85 @@ title: "Configure a yahboom-dofbot Arm"
...
 
-The following attributes are available for yahboom-dofbot arms:
+The following attributes are available for `yahboom-dofbot` arms:
 
-| Name | Inclusion | Description |
-| ---- | --------- | ----------- |
-|  |  |  |
+| Attribute | Type | Inclusion | Description |
+| --------- | ---- | ----------| ----------- |
+| `i2c`  | string | **Required** | The `name` of the Inter-Integrated Circuit (I<sup>2</sup>C) bus on your GPIO [board](/components/board/) where the connection to the `yahboom-dofbot` is made. See [configuration info](/components/board/#i2cs). |
```
Comment (andf-viam) [inline docs/components/arm/yahboom-dofbot.md:87]: Yeah! Nice with `(I<sup>2</sup>C)`!

One of these days I'm gonna `s/I2C/(I<sup>2</sup>C)/g` the whole corpus!

**Review by andf-viam — APPROVED** [review]
Nice! Tiny comments, good stuff!

File: docs/components/arm/yahboom-dofbot.md (hunk 2 above)
Comment (sguequierre) [inline docs/components/arm/yahboom-dofbot.md:81]:
```suggestion
{{< /tabs >}}
```

**Pull request merged**